docs/UserGuide.md

```diff
@@ -51,98 +51,141 @@ Paidlancers is a **desktop app for keeping track of your freelancing events, opt
+
+**Format**: `newrate INDEX AMOUNT`
+
+- Adds a specified rate, `AMOUNT`, to an event at the specified `INDEX`.
+
+  - The `INDEX` refers to the index number in the displayed events list.
+  - The `INDEX` must be a positive integer 1, 2, 3, …​
+
+**Example**: 
+- `newrate 2 100` adds the rate of `100` to the `2nd event` in the event list.
 
 
 ### Marks event as done: `mark`
-Marks a specified event in the address book as done.  
-Format: `mark INDEX`  
-Marks the event at the specified **INDEX** as done.  
+
+Marks a specified event in the address book as done. 
+
+**Format**: `mark INDEX`  
+
+- Marks the event at the specified `INDEX` as done. 
+
 The event must not be marked as done beforehand.  
-Example:  
-- `mark 2` marks the 2nd event as done.
+
+**Example**:  
+- `mark 2` marks the `2nd event` as done.
 
 ### Create a new event: `newevent`
+
 Creates a new event  
-Format: `newevent NAME`  
-Creates a new event with the specified event **NAME**  
-Example: 
+
+**Format**: `newevent NAME`  
+
+- Creates a new event with the specified event `NAME` 
+
+**Example**: 
 - `newevent Singing` creates an event that has the name “Singing”.
 
-Listing all events : listevent
+### Listing all events : `listevent`
+
 Shows a list of all events in the address book
-Format: listevent
+
+**Format**: `listevent`
 
 
-Deleting an event : deleteevent
+### Deleting an event : `deleteevent`
+
 Deletes the specified event from the event book.
-Format: deleteevent INDEX
-Deletes the event at the specified INDEX
-The index refers to the index number shown in the displayed event list.
-The index must be a positive integer 1, 2, 3, …​
-Examples:
-list followed by delete 2 deletes the 2nd event in the event book.
+
+**Format**: `deleteevent INDEX`
+
+- Deletes the event at the specified INDEX
+  - The `INDEX` refers to the index number shown in the displayed event list.
+  - The `INDEX` must be a positive integer 1, 2, 3, …​
+  
+**Examples**:
+  list followed by delete 2 deletes the 2nd event in the event book.
 
 
-Adds dates to an event : adddate
+### Adds dates to an event : `adddate`
+
 Adds a starting time and ending time to the specified event in the event book.
-Format: adddate INDEX /from dd/MM/yy /to dd/MM/yy
-Adds starting and ending time to the event at the specified INDEX
-The index refers to the index number shown in the displayed event list.
-The index must be a positive integer 1, 2, 3, …​
-The event must not have time associated with it beforehand
 
+**Format**: `adddate INDEX /from dd/MM/yy /to dd/MM/yy`
+
+- Adds starting and ending time to the event at the specified INDEX
+    - The `INDEX` refers to the index number shown in the displayed event list.
+    - The `INDEX` must be a positive integer 1, 2, 3, …​
+  
+The event must not have time associated with it beforehand.
+
+
+### Saving all events and contacts: `save`
 
-Saving all events and contacts: save
 Saves all entries in Paidlancer.
-Format: save
+
+**Format**: `save`
```

Comment:
I think we don't have to include the save and load commands because there is autosave.

Reply:
👍